updated the images and content (#2165)
SumoLogic/sumologic-documentation: docs/observability/aws/about.md

### System architecture and monitoring

Sumo Logic provides an AWS CloudFormation templates (CFN) and Terraform scripts that automate the setup and installation of the AWS Observability Solution for your AWS accounts and regions.

* **Sumo Logic AWS Observability** supports the following AWS services: EC2, ECS, RDS, ElastiCache, API Gateway, Lambda, DynamoDB, Classic ELB, Application ELB, Network ELB, AWS SNS, and AWS SQS.

After you have [prepared](/docs/observability/aws/deploy-use-aws-observability/before-you-deploy) and [deployed](/docs/observability/aws/deploy-use-aws-observability) the solution, you can [add and configure additional AWS services](/docs/observability/aws/other-configurations-tools#Add_a_New_AWS_Service_to_the_AWS_Observability_Explore_Hierarchy) as your infrastructure grows.

![AWS_Observ_architecture.jpg](/img/observability/AWS_Observ_architecture.jpeg)

> ### Data Collection and Dashboards

Reviewer: ```suggestion
### Data collection and dashboards
```

This new metadata can also be used in ad-hoc logs and metrics searches.

<img src={useBaseUrl('/img/observability/AWS_Observability_Collection.png')} alt="AWS_Observability_Collection"/>

> You can navigate from overview dashboards of the infrastructure and drill down into account, AWS Region, service, or entity views. The intuitive navigation enables you to quickly resolve issues, minimize downtime, and improve system availability. See [View AWS Observability Solution Dashboards](/docs/observability/aws/deploy-use-aws-observability/view-dashboards) for details.

Reviewer: ```suggestion
You can navigate from overview dashboards of the infrastructure and drill down into account, AWS region, service, or entity views. The intuitive navigation enables you to quickly resolve issues, minimize downtime, and improve system availability. See [View AWS Observability Solution Dashboards](/docs/observability/aws/deploy-use-aws-observability/view-dashboards) for details.
```

* Sumo Logic's AWS Observability API Gateway dashboards provide insights into Amazon API Gateway tasks while accepting and processing concurrent API calls throughout your infrastructure, including traffic management, CORS support, authorization, and access control, throttling, monitoring, and API version management.
* [AWS Observability Application Load Balancer](/docs/observability/aws/integrations/aws-application-load-balancer) - The [AWS Application Load Balancer](https://aws.amazon.com/elasticloadbalancing/) service functions at the application layer, it receives requests, evaluates the listener rules in priority order to determine which rule to apply, and then selects a target from the target group.
* Sumo Logic's AWS Elastic Load Balancing app is a unified logs and metrics App that gives you visibility into the health of your AWS Application Load Balancer and target groups. The app's preconfigured dashboards provide insights into latency, request and host status, threat intel, and HTTP backend codes by Availability Zone and target group.

> * [AWS Observability Host Metrics (EC2)](docs/observability/aws/integrations/aws-ec2-host-metrics) - The Sumo Logic's app for AWS Observability Host Metrics (EC2) allows you to collect local host metrics and displays them using predefined search queries and dashboards for your [Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/) service. App dashboards provide a visual analysis of local host metrics for CPU, Disk, Memory, Network, and TCP.

Reviewer 2: ```suggestion
* [AWS Observability Host Metrics (EC2)](docs/observability/aws/integrations/aws-ec2-host-metrics) - The Sumo Logic app for AWS Observability Host Metrics (EC2) collects local host metrics and displays them using predefined search queries and dashboards for your [Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/) service. App dashboards provide a visual analysis of local host metrics for CPU, Disk, Memory, Network, and TCP.
```
